pkg/command/context.go

```diff
@@ -129,6 +136,7 @@ func initCreateCtxCmd() {
 	createCtxCmd.Flags().BoolVar(&staging, "staging", false, "use CSP staging issuer")
 	createCtxCmd.Flags().StringVar(&endpointCACertPath, "endpoint-ca-certificate", "", "path to the endpoint public certificate")
 	createCtxCmd.Flags().BoolVar(&skipTLSVerify, "insecure-skip-tls-verify", false, "skip endpoint's TLS certificate verification")
+	createCtxCmd.Flags().StringVar(&contextType, "type", "", "type of context to create (mission-control | application-engine | k8s-cluster-endpoint | k8s-local-kubeconfig)")
```

Comment:
Would request comments/suggestion on the context type names

Reply:
let's go with these; we can update in a followup if necessary.